src/qibocal/protocols/characterization/flux_dependence/utils.py

```diff
@@ -172,6 +184,54 @@ def flux_dependence_plot(data, fit, qubit):
     return figures, fitting_report
 
 
+def flux_crosstalk_plot(data, fit, qubit):
+    figures = []
+    fitting_report = ""
```

Comment:
```suggestion
    fitting_report = "No fitting data"
```
In this way the empty table will be removed.
I know that this is weird... in #406 you will simply be able to set it to None.

Reply:
Thanks, I didn't know that. Indeed, using `None` makes more sense.